client/src/app/dialogs/add-task/add-task.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="title">
+  <div class="title-text">
+    <h1 mat-dialog-title>Create Account</h1>
```

Comment:
What account?

Reply:
Yeah, i wrote similar dialogs in Exadel Sandbox. So I copy pasted some parts of it. Forgot to change the title